util/Nginx/entrypoint.sh

```diff
@@ -31,10 +31,16 @@ mkhomedir_helper $USERNAME
 
 chown -R $USERNAME:$GROUPNAME /etc/bitwarden
 cp /etc/bitwarden/nginx/default.conf /etc/nginx/conf.d/default.conf
-mkdir -p /etc/letsencrypt
-chown -R $USERNAME:$GROUPNAME /etc/letsencrypt
-mkdir -p /etc/ssl
-chown -R $USERNAME:$GROUPNAME /etc/ssl
+if ! gosu $USERNAME:$GROUPNAME test -r /etc/letsencrypt/live
```

Comment:
Why are you checking `/etc/letsencrypt/live` instead of just `/etc/letsencrypt`?

Reply:
The letsencrypt folder is world readable by default, while its contents are readable for the owner and group only